src/core/Notification/Notification.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { Component, forwardRef, ReactNode } from 'react';
+
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import {
+  HtmlDiv,
+  HtmlDivWithRef,
+  HtmlButton,
+  HtmlButtonProps,
+  HtmlDivWithRefProps,
+} from '../../reset';
+import { hLevels } from '../../reset/HtmlH/HtmlH';
+import { Icon } from '../Icon/Icon';
+import { Heading } from '../Heading/Heading';
+
+import { AutoId } from '../utils/AutoId/AutoId';
+import { getConditionalAriaProp } from '../../utils/aria';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
+import { baseStyles } from './Notification.baseStyles';
```

Comment:
1. We prefer not having empty lines between imports.

2. And we would like to have imports ordered as how much much we have to traverse back; first with the longest e.g `../../` then with lower `../`

So the end in this case would be like

```tsx
import { getConditionalAriaProp } from '../../utils/aria';
import { Icon } from '../Icon/Icon';
import { Heading } from '../Heading/Heading';
import { AutoId } from '../utils/AutoId/AutoId';
import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
import { baseStyles } from './Notification.baseStyles';
```

Reply:
Fixed. Removed empty spaces.

Reply:
Nitpicking here, but it seems that `import { getConditionalAriaProp } from '../../utils/aria';` is not as suggested in step 2; still in old order.

Reply:
Fixed. Changed order of imports.